calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php

```diff
@@ -56,7 +56,7 @@ public function generate()
 			$objTemplate->title = $this->headline;
 			$objTemplate->id = $this->id;
 			$objTemplate->link = $this->name;
-			$objTemplate->href = 'contao/main.php?do=themes&amp;table=tl_module&amp;act=edit&amp;id=' . $this->id;
+			$objTemplate->href = StringUtil::specialcharsUrl(System::getContainer()->get('router')->generate('contao_backend', array('do'=>'themes', 'table'=>'tl_module', 'act'=>'edit', 'id'=>$this->id)));
```

Comment:
Why did you wrap all the routes with `StringUtil::specialcharsUrl()`? The `replaceOldBePaths()` method does not do this, so either the call is unnecessary or it should be added to the method as well, shouldn‘t it?

Reply:
the old URLs all encoded `&` to `&amp;` and I guessed we should continue to do that 🤷

Reply:
@ausi WDYT?

Reply:
Yes, the URL has to be HTML-encoded IMO as the template expects that.